packages/styletron-core/src/index.js

```diff
@@ -59,7 +61,8 @@ class StyletronCore {
       return cached;
     }
     const virtualCount = this.incrementVirtualCount();
-    const hash = virtualCount.toString(36);
+    const readableClassName = decl.prop + '-' + decl.val;
```

Comment:
I've just realized this, but this mode will produce collisions that won't occur with the standard, sequential class names.

This will happen with media queries and pseudo classes, for example:

```js
styletron.injectDeclaration({color: 'red'});
// → 'a' (standard mode)
// → 'color-red' 
styletron.injectDeclaration({color: 'red', media: 'screen'});
// → 'b' (standard mode)
// → 'color-red'
styletron.injectDeclaration({color: 'red', pseudo: ':hover'});
// → 'c' (standard mode)
// → 'color-red'
```

A couple of possibilities:
1) *Always* use standard, sequential class names in cases of declarations with `psuedo` and/or `media` attributes.
2) Have additional prefixes based on `psuedo` or `media` properties. The trick here is we need a bijection from these values to valid class name identifiers.